include/control_toolbox/pid_ros.hpp

```diff
@@ -113,6 +128,13 @@ class PidROS
    */
   void reset();
 
+  /*!
+   * \brief Reset the state of this PID controller
+   *
+   * \param save_iterm boolean indicating if integral term is retained on reset()
+   */
+  void reset(bool save_iterm = false);
```

Comment:
```suggestion
  void reset(bool save_iterm);
```

It is better to remove the default arg, I'm not sure how the compiler can differentiate between the non parsed one and the default param one

Reply:
it did not complain although I have used both versions in the tests. but it is not really necessary here.